src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/ICertificatePal.cs

```diff
@@ -10,6 +10,16 @@
 
 namespace Internal.Cryptography
 {
+    internal struct PolicyData
+    {
+        public byte[] ApplicationCertPolicies;
+        public byte[] CertPolicies;
+        public byte[] CertPolicyMappings;
+        public byte[] CertPolicyConstraints;
+        public byte[] EnhancedKeyUsage;
+        public byte[] InhibitAnyPolicyExtension;
```

Comment:
Please  change all of these to say `internal` (They're effectively internal because the type is, but any time I see `public` my immediate feeling is "really?!".